### PR TITLE
feat(api): Add Organization level issues resolved in release endpoint.

### PR DESCRIPTION
**Pull request by wedamija**
Refactored the project level endpoint to share code. Reduced duplication in tests.

Fixes APP-942

### CODE REVIEW

File: src/sentry/api/bases/release.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+from sentry.api.exceptions import ResourceDoesNotExist
+from sentry.models import (
+    GroupLink,
+    GroupResolution,
+    Release,
+    ReleaseCommit,
+)
+
+
+class IssuesResolvedInReleaseMixin(object):
```
Comment (markstory) [inline src/sentry/api/bases/release.py]: Does this need to be a mixin or an instance method? Seems like something that could be a module function avoiding inheritance entirely.

Comment (wedamija) [reply src/sentry/api/bases/release.py]: Seems reasonable, any thoughts on where we should put it? I don't have a great feel for where these sorts of functions could go, and don't really want to just keep dumping things into utils.

Comment (markstory) [reply src/sentry/api/bases/release.py]: Could be in this module.

Comment (wedamija) [reply src/sentry/api/bases/release.py]: Discussed on slack, decided to put in `api/helpers/releases.py`

File: src/sentry/api/endpoints/organization_issues_resolved_in_release.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+        groups = Group.objects.filter(
+            project_id__in=self.get_project_ids(request, organization),
+            id__in=group_ids,
+        )
+
+        context = serialize(
+            list(groups),
+            request.user,
+            GroupSerializer(
+                environment_func=self._get_environment_func(request, organization.id)
```
Comment (macqueen) [inline src/sentry/api/endpoints/organization_issues_resolved_in_release.py]: is this going to need to support multiple environments?

**Review by macqueen — APPROVED** [review]

File: src/sentry/api/endpoints/organization_issues_resolved_in_release.py (hunk 2 above)
Comment (wedamija) [reply src/sentry/api/endpoints/organization_issues_resolved_in_release.py]: Hmm, I think it is. This seems relevant to a couple of places? I'll make a ticket for handling this separately

Comment (wedamija) [reply src/sentry/api/endpoints/organization_issues_resolved_in_release.py]: https://getsentry.atlassian.net/browse/APP-956

Comment (macqueen) [reply src/sentry/api/endpoints/organization_issues_resolved_in_release.py]: I added a `GroupSerializer` that supports multiple envs / uses snuba only: https://github.com/getsentry/sentry/blob/master/src/sentry/api/serializers/models/group.py#L550

but yeah, fine to do as a follow up if you prefer!

Comment (wedamija) [reply src/sentry/api/endpoints/organization_issues_resolved_in_release.py]: Oh right, will look into using that as part of this. Thanks!

**Pull request merged**